r/R/arrow-datum.R

```diff
@@ -46,6 +46,22 @@ as.vector.ArrowDatum <- function(x, mode) {
   )
 }
 
+#' @export
+na.omit.ArrowDatum <- function(object, ...){
+  object$Filter(!is.na(object))
+}
+
+#' @export
+na.exclude.ArrowDatum <- na.omit.ArrowDatum
+
+#' @export
+na.fail.ArrowDatum <- function(object, ...){
+  if(object$null_count > 0){
+    stop("missing values in object")
```

Comment:
One nit on spacing around `if()` to match our existing code style, and we always do `call. = FALSE` on `stop()`

```suggestion
  if (object$null_count > 0) {
    stop("missing values in object", call. = FALSE)
```